ui/src/__tests__/pages/domain/[domain]/service/[service]/instance/dynamic.test.js

```diff
@@ -76,10 +76,12 @@ describe('DynamicInstancePage', () => {
             },
         };
         let serviceHeaderDetails = {
-            description:
-                'Here you can add / see instances which can not obtain Athenz identity because of limitations, but would be associated with your service.',
-            url: '',
-            target: '_blank',
+            dynamic : {
+                description:
+                    'Here you can see instances which are running with this service identity',
```

Comment:
@chandrasekhar1996 is this what you meant to change?

Reply:
yes